lib/struct.js

```diff
@@ -398,7 +395,6 @@ module.exports = function() {
         extractShort: extractShort,
         extractFloat: extractFloat,
         extractByte: extractByte,
-        extractSByte: extractSByte,
```

Comment:
Looks like you're not exporting it anymore? Intentional?

Also there are tests for all the stuff in struct, I believe. Shouldn't you add a couple?